pages/login.page.js

```diff
@@ -0,0 +1,29 @@
+const PAGE_TIMEOUT = 10000;
+
+class LoginPage {
+    
+    get usernameInput () { return $('#username'); }
+    get passwordInput () { return $('#password'); }
+    get signInButton () { return $('#submit'); }
+    get rememberMe () { return $('#remember-me'); }
+    get alert () {return $('[class= "badge badge-pill badge-danger"]'); }
```

Comment:
This is not a very descriptive name. Firstly, it can be confused with a javascript alert (that is a pop up message that appears at the top of a page typically), secondly, it's a bit too general, as it doesn't hint about its contents: Prefer something that points to its function on the page (like: `failedLoginMessage`) or something like that